tests/vmi_networking_test.go

```diff
@@ -748,6 +748,9 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					Expect(tests.PingFromVMConsole(clientVMI, "google.com")).To(Succeed())
 				}
 
+				By(fmt.Sprintf("Checking ping from clientVMI console to serverVMI(%s)", serverIP))
```

Comment:
i think we dont like, nor need redundant prints, in case it fails we we see in the expecter dump.

Reply:
I am following stuff like `By("Connecting from the client vm")` when you are real time debugging stuff with `-ginkgo.v` is nice to see what's doing when you see it's taking a lot of time.

Reply:
yes but Jed added expecter dump in case of failure (or even in case of `.v` without failure)
which already shows the ip it actually ping,
i understand it sometimes nicer at the `By`
its just will be twice, 

up to you then

Reply:
@qinqon your reasoning is good for debugging a specific thing, not when I just want to know what is wrong (99% of the time). Adding text to code and logs all over is not productive in a common run and when I need to read the code it makes me read it with no added value. I prefer prints/logs limited to what is needed and not all over.

But you insist I will not block on this nit.

Reply:
Ack no problem will remove it.

Reply:
Done.